src/open_inwoner/configurations/migrations/0058_customfontset.py

```diff
@@ -0,0 +1,68 @@
+# Generated by Django 3.2.23 on 2024-01-23 15:38
+
+import django.core.validators
+from django.db import migrations, models
+import django.db.models.deletion
+import open_inwoner.configurations.models
+import open_inwoner.utils.files
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("configurations", "0057_siteconfiguration_theme_stylesheet"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="CustomFontSet",
+            fields=[
+                (
+                    "id",
+                    models.AutoField(
+                        auto_created=True,
+                        primary_key=True,
+                        serialize=False,
+                        verbose_name="ID",
+                    ),
+                ),
+                (
+                    "text_body_font",
+                    open_inwoner.configurations.models.CustomFontField(
+                        blank=True,
+                        help_text="Regular font for the text body. Supported are files with the .ttf extension.",
```

Comment:
Minor detail and not disapproved, but the message here may need to be more in 'active' language, something like "Upload body text font here. TTF font types only" - same as the one for the Headings.
Also I am wondering if we want this font upload to be all the way in the Admin bottom, instead of higher up, closer to the 'custom colors' section?

Reply:
I changed the messages, but moving the admin inline seems to require modifications to the admin template. Let's discuss and perhaps create a separate task for this.

Reply:
@pi-sigma to add to that: the amount of fields on the configuration singleton is getting a bit out of hand, so for that discussion we could also consider splitting the model, like in technical system configuration, content options, layout stuff and texts. 

This could also make progress to getting rid of that crazy context-processor that just copies a pile of data from the singleton into a dictionary.

Reply:
Sounds good. We should come up with a good division though, lest we end up proliferating configuration types and creating something worse.